.pipelines/build.yml

```diff
@@ -0,0 +1,653 @@
+# 'Allow scripts to access the OAuth token' was selected in pipeline.  Add the following YAML to any steps requiring access:
+#       env:
+#           MY_ACCESS_TOKEN: $(System.AccessToken)
+# Variable 'MajorVersion' was defined in the Variables tab
+# Variable 'MinorVersion' was defined in the Variables tab
+trigger:
+  branches:
+    include:
+    - refs/heads/master
+  batch: True
+schedules:
+- cron: 0 18 * * *
+  branches:
+    include:
+    - refs/heads/master
+name: $(MajorVersion).$(MinorVersion).$(date:yyMMdd)$(rev:.r)
+jobs:
+- job: BuildBinaries
+  pool:
+    name: Azure Pipelines
+    vmImage: 'windows-2022'
+    demands:
+    - msbuild
+  strategy:
+    matrix:
+      x86:
+        buildPlatform: 'x86'
+      x64:
+        buildPlatform: 'x64'
+      arm:
+        buildPlatform: 'arm'
+      arm64:
+        buildPlatform: 'arm64'
+  steps:
+  - checkout: self
+    clean: true
+    persistCredentials: True
+
+  - task: NuGetToolInstaller@1
+    displayName: Use NuGet 6.0.2
+    continueOnError: True
+    inputs:
+      versionSpec: 6.0.2
+  - task: NuGetCommand@2
+
+    displayName: NuGet restore
+
+  - task: CmdLine@2
+    displayName: Build Tools
+    inputs:
+      script: |
+        if "%VSCMD_VER%"=="" (
+            pushd c:
+            call "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\Tools\VsDevCmd.bat" >nul 2>&1
+            popd
+        )
+
+        build_test_all.cmd $(BuildPlatform) $(BuildConfiguration) $(Build.BuildNumber) true
+      failOnStderr: true
+
+  - task: ComponentGovernanceComponentDetection@0
+    displayName: Component Detection
+    condition: eq(variables['BuildPlatform'], 'x64')
+
+  - task: PublishTestResults@2
+    displayName: Publish Test Results
+    enabled: False
+
+  - task: CopyFiles@2
+    displayName: Stage cppwinrt.*
+    condition: and(eq(variables['BuildPlatform'], 'x86'), eq(variables['BuildConfiguration'], 'release'), in(variables['Build.Reason'], 'Manual'))
+    inputs:
+      SourceFolder: $(Build.SourcesDirectory)\_build\$(BuildPlatform)\$(BuildConfiguration)
+      Contents: |
+        cppwinrt.exe
+        cppwinrt.pdb
+      TargetFolder: $(Build.ArtifactStagingDirectory)\cppwinrt
+
+  - task: CopyFiles@2
+    displayName: Stage Component cppwinrtvisualizer.*
+    condition: and(or(eq(variables['BuildPlatform'], 'x86'), eq(variables['BuildPlatform'], 'x64'), eq(variables['BuildPlatform'], 'arm64')), eq(variables['BuildConfiguration'], 'release'), in(variables['Build.Reason'], 'Manual'))
+    inputs:
+      SourceFolder: '$(Build.SourcesDirectory)\natvis\$(BuildPlatform)\$(BuildConfiguration)\Component '
```

Comment:
The sourceFolder string has an extra space at the end.  Also the quotes on it are mismatched compared to all other CopyFiles tasks that do not use quotes for the path.

Reply:
> The sourceFolder string has an extra space at the end. Also the quotes on it are mismatched compared to all other CopyFiles tasks that do not use quotes for the path.

Good catch. There was an extra trailing space in the classic pipeline as well. I suppose the YAML exported tried to preserve it and put the whole thing in quotes.